core/qt-ble.cpp

```diff
@@ -366,13 +366,15 @@ dc_status_t BLEObject::setupHwTerminalIo(QList<QLowEnergyCharacteristic> allC)
 	return setHwCredit(MAXIMAL_HW_CREDIT);
 }
 
+#if !defined(Q_OS_WIN)
 // Bluez is broken, and doesn't have a sane way to query
 // whether to use a random address or not. So we have to
 // fake it.
 static int use_random_address(dc_user_device_t *user_device)
 {
 	return IS_SHEARWATER(user_device) || IS_GARMIN(user_device);
 }
+#endif
```

Comment:
hi @salvadorcunat 
i think this PR picked some extra commits.
could you please double check?

Reply:
hi @neolit123.
This is not for master, but for fixTravisWindows branch, where @dirkhh is testing new mxe binaries.
It's the first time I send anything to a branch other than master, I think I've made something wrong while building the PR. Does this have any sense?
When I saw the PR last night against the fixTravisWindows branch last night, it was only my commit without Dirk's.

Reply:
Hi @neolit123 .
I've followed your git workflow and it seens to have worked.  Please take a look, I think the PR is correct now.
Best regards.